Backend/WebApi.Weather/LocationData.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace WebApi.Weather
+{
+    /// <summary>
+    /// Returned by Geocoding Api.
```

Comment:
Jak to jest zwracane przez jakieś API to zmień class na record, to nie pozwoli na modyfikowanie odpowiedzi.

Reply:
w naszym przypadku nie potrzebne usunąłem